libraries/helpers.rb

```diff
@@ -20,9 +20,11 @@ module Helpers
     include Chef::Mixin::ShellOut
 
     require 'securerandom'
+    require 'tempfile'
 
     def psql_command_string(new_resource, query, grep_for: nil, value_only: false)
-      cmd = "/usr/bin/psql -c \"#{query}\""
+      query_file = Tempfile.create('sql-query') { |f| f << query }
+      cmd = "/usr/bin/psql -f #{query_file.path}"
```

Comment:
How come we need the Tempfile here?

Reply:
It is an attempt at getting around quoting issues when you try to install an extension with a hypen in it like `uuid-ossp`. Sadley it didn't work at all

Reply:
`%w()` does seem like the right direction in this case